LibreNMS/Cache/Device.php

```diff
@@ -46,7 +46,7 @@ public function getPrimary() : \App\Models\Device
      *
      * @param int $device_id
      */
-    public function setPrimary($device_id)
+    public function setPrimary(int $device_id)
```

Comment:
Shouldn't this one be ?int too?

Reply:
Not really since it makes no sense it being null i suppose?

Reply:
Also, we don't want this to be null? The other function is a feature. Here I would kinda call it a bug? Better to be strict then

Reply:
Oopsie, got that mixed up with something else :)